### PR TITLE
toolchain/fortify-headers: update download url

### PR DESCRIPTION
**Pull request by zpc0**
Development of fortify-headers has been moved to GitHub. Gentoo [1] and Alpine Linux [2] already uses it.

Since updating fortify-headers to new version causes a lot of compile errors, this commit does not bump version.

[1]: https://packages.gentoo.org/packages/sys-libs/fortify-headers
[2]: https://git.alpinelinux.org/aports/log/main/fortify-headers

### CODE REVIEW

File: toolchain/fortify-headers/Makefile [hunk 1]
```diff
@@ -11,9 +11,9 @@ PKG_NAME:=fortify-headers
 PKG_VERSION:=1.1
 PKG_RELEASE=3
 
-PKG_SOURCE_URL:=https://dl.2f30.org/releases
+PKG_SOURCE_URL:=https://codeload.github.com/jvoisin/$(PKG_NAME)/tar.gz/refs/tags/$(PKG_VERSION)?
```
Comment (robimarko) [inline toolchain/fortify-headers/Makefile:14]: I am not a fan of using these autogenerated GH releases at all.
Also, simply using tags does not guarantee reproducibility as tags can be changed upstream.

So, if you want to use the new upstream, I would suggest pointing to a specific hash instead and letting the build system package the tarball instead.

File: toolchain/fortify-headers/Makefile [hunk 2]
```diff
@@ -8,12 +8,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/target.mk
 
 PKG_NAME:=fortify-headers
-PKG_VERSION:=1.1
```
Comment (robimarko) [inline toolchain/fortify-headers/Makefile:11]: You can leave PKG_VERSION if it is indeed 1.1, PKG_SOURCE_VERSION just points to its hash

**Comment by zpc0** [comment]
@robimarko Thank you for review! I updated commit based on your suggestion.

**Comment by robimarko** [comment]
The PKG_MIRROR_HASH is incorrect.

Do you have tools/zstd built?

**Comment by zpc0** [comment]
> The PKG_MIRROR_HASH is incorrect.
> 
> Do you have tools/zstd built?

I forgot updating mirror hash when I restored PKG_VERSION. Sorry.
I fixed PKG_MIRROR_HASH.

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**